src/help/index.ts

```diff
@@ -202,10 +204,14 @@ export class Help extends HelpBase {
       command.id = command.id.replace(/:/g, this.config.topicSeparator)
       command.aliases = command.aliases && command.aliases.map(a => a.replace(/:/g, this.config.topicSeparator))
     }
-    const help = new this.CommandHelpClass(command, this.config, this.opts)
+    const help = this.getCommandHelpClass(command)
```

Comment:
Do we really need a whole new method for this if this is the only place it's used?

Reply:
@RodEsp This function will be overridden in the custom help class to allow the determination of short help.